index.js

```diff
@@ -0,0 +1,36 @@
+// Add a footer 
+const footer = document.createElement("footer");
+const body = document.querySelector("body");
+body.appendChild(footer);
```

Comment:
instead of referencing the variable, use
`document.body.appendChild(footer);`, which will show the footer